meshtastic/tests/test_api_compat.py

```
@@ -0,0 +1,161 @@
+    for method in required_methods:
+        assert hasattr(MeshInterface, method), f"MeshInterface.{method} must exist"
+
+
+# ---------------------------------------------------------------------------
+# Module-level constants
+# ---------------------------------------------------------------------------
+
+
+def test_node_exports_constants() -> None:
+    """Exported constants must be present and be positive integers."""
+    assert isinstance(MAX_CHANNELS, int) and MAX_CHANNELS > 0
```

Ruff (0.15.7)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)